test/goldens/end2end/audio_service-0.18.10.json_report.md

```diff
@@ -20,11 +20,11 @@ Detected license: `MIT`.
 
 ## 20/20 Platform support
 
-### [*] 20/20 points: Supports 4 of 6 possible platforms (**iOS**, **Android**, **Web**, Windows, **macOS**, Linux)
```

Comment:
I think you need to rebase the PR on HEAD.

Reply:
Ah thanks! - I was getting very confused